bootstrap.py

```
@@ -90,9 +90,15 @@ def run_command(
                 text=True,
                 stdout=handle,
                 stderr=subprocess.STDOUT,
+                shell=os.name == "nt",
             )
     else:
-        result = subprocess.run(resolved, cwd=str(cwd) if cwd else None, text=True)
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)